src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,9 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    public static final int kMotorOneID = 11;
```

Comment:
Remember all of the subsystems will use the Constants file. How can we name the motors that belong to intake so they're easy to identify?